source/content/supported-drupal.md

```diff
@@ -111,6 +111,16 @@ When updates are available, Drupal's Update Status module will notify you of the
 
 ![Tag1 D7ES available updates](../images/supported-drupal-d7es-update-status-security-alert.png)
 
+<Alert title="Data retention policy"  type="info" >
+
+##### Requirement for maintaining contrib module LTS coverage
```

Comment:
```suggestion
##### Functioning cron is necessary for maintaining contrib module LTS coverage
```

We should maybe rewrite the following paragraph to lead with the action necessary instead of leading with the explanation and build to to the "so what"